open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/index.html

```diff
@@ -0,0 +1,102 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
```

Comment:
We need to start adding copyright to all files

Reply:
done

Reply:
There's also pom.xml -- though I see no precedence in the top level pom, on that basis no reason to block this PR for now. We probably need to scan/act on any report and fix up globally in the tree. will discuss/raise seperate issue